src/quilla/ctx.py

```diff
@@ -77,7 +80,9 @@ def __init__(
         is_file: bool = False,
         no_sandbox: bool = False,
         definitions: List[str] = [],
-        logger: Optional[Logger] = None
+        logger: Optional[Logger] = None,
+        run_id: Optional[str] = None,
```

Comment:
Any chance we can set run id default value here?

Reply:
The run ID default is set later on, it just needs to be randomized so it’s better to do it explicitly

Reply:
By that i mean, by setting it on the constructor there can often be some weird variable caching issues when you have objects instead of literals as part of the defaults when creating multiple objects. Setting it below on the body of the function prevents that from happening